### ISSUE TITLE
ERROR: Could not find a version

### ISSUE THREAD

**Issue by shawnngtq** [joeyism/linkedin_scraper#50]
I am unable to download this package. I tried:

```
conda install linkedin_scraper
pip install linkedin_scraper
pip3 install --user linkedin_scraper
```

These are the errors I am seeing:
```
ERROR: Could not find a version that satisfies the requirement request (from linkedin_scraper) (from versions: none)
ERROR: No matching distribution found for request (from linkedin_scraper)
```

@joeyism, can't wait to try it out 😁

**Comment by dfreire-dot** [comment]
I have the same error. Did you find any solution? I haven't found anything that solved it for me

**Comment by jiteshgurav** [comment]
same, has this library been taken off pypi?

**Comment by shawnngtq** [comment]
No solution from me, I am still waiting for help too ...

**Comment by joeyism** [comment]
I fixed it in version [2.5.1](https://github.com/joeyism/linkedin_scraper/releases/tag/2.5.1). I tried it locally, it should be working now. If you're getting the same error, you may need to force upgrade with
```bash
pip3 install --user linkedin_scraper --upgrade
```

**Comment by shawnngtq** [comment]
@joeyism thanks for the quick fix. I confirmed it works for me.
